packages/scan/src/models/Severity.spec.ts

```diff
@@ -38,7 +38,8 @@ describe('Severity', () => {
     it.each([
       { input: Severity.LOW, expected: 1 },
       { input: Severity.MEDIUM, expected: 2 },
-      { input: Severity.HIGH, expected: 3 }
+      { input: Severity.HIGH, expected: 3 },
+      { input: Severity.CRITICAL, expected: 4 }
```

Comment:
You have to add test data for `severityComparator`as well to cover missing cases